src/Storage/Storage/Blob/Cmdlet/NewAzureStorageBlobSasToken.cs

```diff
@@ -28,7 +28,9 @@ namespace Microsoft.WindowsAzure.Commands.Storage.Blob.Cmdlet
     using global::Azure.Storage.Blobs;
     using System.Collections.Generic;
     using global::Azure.Storage;
+    using Microsoft.WindowsAzure.Commands.Common.CustomAttributes;
 
+    [GenericBreakingChange("SAS token created will not contain a leading question mark ? in a future release.")]
```

Comment:
How about:

The leading question mark '?' of the created SAS token will be remove in a future release.

Reply:
Done

Reply:
Done